### ISSUE TITLE
[Enhancement] Support for TensorBoard?

### ISSUE THREAD

**Issue by NeoChen1024** [derrian-distro/LoRA_Easy_Training_Scripts#245]
Is it possible to have support for TensorBoard to log learning rate & loss? It would be helpful to finetune hyper-parameters for Prodigy and Adafactor optimizers

**Comment by Jelosus2** [comment]
It already supports tensorboard, refer to the logging args section.

**Comment by NeoChen1024** [comment]
Sorry, my bad...
